lib/blue_bird/group.ex

```diff
@@ -0,0 +1,17 @@
+defmodule BlueBird.Group do
+  @moduledoc """
+  Defines the `Group` struct.
+  """
+  defstruct [
+    :name,
+    :description
+  ]
+
+  @typedoc """
+  Type that represents the Group struct.
+  """
+  @type t :: %BlueBird.Group{
+    name: String.t | nil,
+    description: String.t | nil
```

Comment:
The macro sets the description to `""` if it is not set, so the nil can be removed here as well.